src/Microsoft.AspNet.Mvc.ViewFeatures/ViewFeatures/ViewExecutor.cs

```diff
@@ -62,6 +62,7 @@ public static class ViewExecutor
                     htmlHelperOptions);
 
                 await view.RenderAsync(viewContext);
+                await writer.FlushAsync();
```

Comment:
This deserves a comment explaining why this is necessary, and at least a unit test. /grumpy